process/factory/shard/preProcessorsContainerFactory_test.go

```diff
@@ -350,7 +366,9 @@ func TestPreProcessorsContainerFactory_CreateErrScrPreproc(t *testing.T) {
 
 	args := createMockPreProcessorsContainerFactoryArguments()
 	args.DataPool = dataPool
-	args.SmartContractResultPreProcessorCreator, _ = preprocess.NewSmartContractResultPreProcessorFactory()
+	runTypeComps := processMocks.NewRunTypeComponentsStub()
```

Comment:
You don't need to do this. `createMockPreProcessorsContainerFactoryArguments` already does this and `SCResultsPreProcessorFactory` is already initialized with `NewSmartContractResultPreProcessorFactory`

Reply:
Right, deleted completely